osmaxx-py/osmaxx/excerptexport/models/extraction_order.py

```diff
@@ -28,6 +51,7 @@ class ExtractionOrder(models.Model):
     orderer = models.ForeignKey(User, related_name='extraction_orders', verbose_name=_('orderer'))
     excerpt = models.ForeignKey(Excerpt, related_name='extraction_orders', verbose_name=_('excerpt'))
     progress_url = models.URLField(verbose_name=_('progress URL'), null=True, blank=True)
+    download_status = models.IntegerField(_('file status'), choices=DOWNLOAD_STATUSES, default=DOWNLOAD_STATUS_UNKNOWN)
```

Comment:
Because it is common to do it like this in [Django](https://docs.djangoproject.com/en/1.9/ref/models/fields/#choices).